legend-engine-xt-persistence-cloud-pure/src/main/resources/core_persistence_cloud/external/persistence/aws/metamodel.pure

```diff
@@ -55,3 +57,12 @@ meta::external::persistence::aws::metamodel::AwsPartition
    {doc.doc = 'AWS GovCloud (US) regions'}
    AWS_US_GOV
 }
+
+Class meta::external::persistence::aws::metamodel::connection::AwsFinCloudConnection extends Connection
```

Comment:
This connection type is unlike the other connection types.

We have types for connections that model traditional databases - e.g RelationalConnection 
We have types for connections that model APIs - e.g ServiceStoreConnection 
We have types for connections that model blobs of data - e.g JsonModelConnection 

Not sure how to classify this connection.

Reply:
We should come with a strategy similar as relational database connection, such that we only have a single top level connection type, and use connection specification and auth strategy to capture the different details of the connection.

Reply:
I'm thinking of setting up a PersistenceConnection that we can use for the time being. Subtypes would be S3Connection, KafkaConnection, and any others we'll be using to specify values for external systems interpreting a Persistence specification.

Goal would be to replace with the top-level connection Rafael mentions once available. Each can evolve to a full-fledged connection if/when we model a Store and start to involve in execution.

LMK what you think @epsstan